documentation/assemblies/upgrading/assembly-api-conversion.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+
+[id='assembly-api-conversion-{context}']
+= Converting Strimzi custom resources to the v1 API
+
+[role="_abstract"]
+This section describes how to prepare Strimzi custom resources and CRDs for the `v1` API before performing an upgrade.
+
+Prepare Strimzi custom resources and CRDs for the `v1` API by using the Strimzi API conversion tool or by applying the required changes manually.  
+The tool is provided with the release artifacts.  
+To use the tool, extract it and run the conversion script from the `bin` directory.
+
+The migration to the `v1` API has two stages.
+
+* Version 0.49 introduces the `v1` API and deprecates the older `v1beta2` version.
```

Comment:
Just a nit. But we have these two old versions there as well for topics and users.

```suggestion
* Version 0.49 introduces the `v1` API and deprecates the older `v1alpha1`, `v1beta1`, and `v1beta2` versions (`v1alpha1` and `v1beta1` versions are used only for `KafkaUser` and `KafkaTopic` resources).
```